OpenKeychain/src/main/java/org/sufficientlysecure/keychain/service/KeyserverSyncAdapterService.java

```diff
@@ -74,6 +77,10 @@
     private static final String ACTION_CANCEL = "cancel";
 
     private AtomicBoolean mCancelled = new AtomicBoolean(false);
+    Context context;
+
+    //zero argument constructor
+    public KeyserverSyncAdapterService(){}
```

Comment:
Doesn't Java supply a zero argument constructor by default if you haven't specified any other constructor? Shouldn't be required now that you removed the other constructor that was here earlier.